hazelcast/src/main/java/com/hazelcast/config/LoginModuleConfig.java

```diff
@@ -99,7 +99,11 @@ public LoginModuleConfig setProperties(Properties properties) {
     }
 
     public LoginModuleConfig setProperty(String key, String value) {
-        properties.setProperty(key, value);
+        if (value != null) {
```

Comment:
This behaviour (`null` value removing property mapping) should be documented in method's javadoc and tested in `LoginModuleConfigTest`. I guess the behaviour change is not a problem, since it was previously throwing `NullPointerException`.

Reply:
Good point. I'll probably add another method `setOrClear(String, String)`, and revert the behavior of the `setProperty` as there could be users relying on the original behavior as a fail-fast approach for login misconfiguration.